ow_lander/scripts/state_publisher.py

```diff
@@ -36,6 +39,15 @@ def _handle_joint_states(self, data):
       rospy.logerr_throttle(1, 'StatePublisher: Array size mismatch')
       return
 
+    dur = data.header.stamp - self._joint_acc.header.stamp
+    dur_float = dur.to_sec() + dur.to_nsec() / 1000000000
```

Comment:
I think you can get away with just `to_sec`
```suggestion
    dur_float = dur.to_sec()
```
_Source_ http://wiki.ros.org/rospy/Overview/Time#Converting_Time_and_Duration_instances

Reply:
Thanks. You found my bug. I saw other examples that made me think I had to sum these two values. Now durations are 0.02 as expected.

Reply:
Fixed